content/en/docs/ops/ambient/usage/waypoint/index.md

```diff
@@ -5,5 +5,223 @@ weight: 2
+- Increased granularity: You can apply different policies to different workloads based on their service account.
+- Reduced resource consumption: You only need to deploy a waypoint proxy for each service account that has workloads that require L7 routing or policy enforcement.
+
+However, deploying waypoint proxies at the service account level also has some drawbacks, including:
+
+* Increased complexity: Managing policies at the service account level can be more complex, especially if you have a large number of service accounts.
+* Reduced caching: Waypoint proxies cannot cache routing and policy information at the service account level, which can reduce performance.
+
+### How to choose the right scope for your waypoint proxies
+
+The best scope for your waypoint proxies will depend on your specific needs and requirements. If you have a simple application with a small number of workloads, then namespace-level scope may be a good choice. However, if you have a more complex application with a large number of workloads or if you need to apply different policies to different workloads, then service account-level scope may be a better choice.
+
+Here are some factors to consider when choosing the scope for your waypoint proxies:
+* The number of workloads in your application
+* The complexity of your application
+* The need to apply different policies to different workloads
+* The performance requirements of your application
+* The resource requirements of your application
+
+## Functional overview
+
+The functional behavior of the waypoint proxy is dynamically configured by Istio to serve your applications configurations. This section takes a brief look at these functional aspects - detailed description of the internal design of the waypoint proxy is out of scope for this guide. The detailed functional overview from the Secure Overlay Networking was already discussed in the [Ztunnel L4 Networking Guide](https://deploy-preview-13635--preliminary-istio.netlify.app/latest/docs/ops/ambient/usage/ztunnel/#functionaloverview) hence this section only focuses on functionalities and features that waypoint proxy provides.
+
+{{< image width="100%"
+link="waypoint-architecture.png"
+caption="Waypoint architecture"
+>}}
+
+What is unique about the waypoint proxy is that it runs either per-namespace (default) or per-service account. By running outside of the application pod, a waypoint proxy can install, upgrade, and scale independently from the application pod, providing a centralized approach to managing L7 traffic and enforcing policies as well as reduce operational costs.
+
+Upon deployment of a gateway resource using the `kubectl apply` command, Istio's control plane, IstioD, assumes the role of the waypoint controller. Recognizing the gateway resource with the "istio.io/waypoint" gateway class name, istiod automatically deploys the waypoint proxy based on the specified configuration in the gateway resource.
```

Comment:
I think the docs usually have istiod: https://github.com/istio/istio.io/blob/cf7f060c5c0f50eb187685bdb6638b139f147e65/content/en/docs/reference/glossary/istiod.md?plain=1#L6 

```suggestion
Upon deployment of a gateway resource using the `kubectl apply` command, Istio's control plane, istiod, assumes the role of the waypoint controller. Recognizing the gateway resource with the "istio.io/waypoint" gateway class name, istiod automatically deploys the waypoint proxy based on the specified configuration in the gateway resource.
```